test/com/linkedin/drelephant/BackfillTest.java

```diff
@@ -374,14 +381,13 @@ public void run() {
             "mapreduce_job_3");
         assertAppResultEntry("application_1526555215992_0008", 1526900000000L, 1526900000050L,
             "spark_job_3");
-        // Verify whether jobs were backilled based on lowest ts of apps retrieved from RM and backfill ts in DB.
-        long lowestTsFromRM = 1526800000050L;
-        ElephantBackfillFetcher[] backfillFetchers =
-            new ElephantBackfillFetcher[] {mapreduceBackfillFetcher, sparkBackfillFetcher, tezBackfillFetcher};
-        for (ElephantBackfillFetcher backfillFetcher : backfillFetchers) {
+        DummyElephantBackfillFetcher[] backfillFetchers =
+            new DummyElephantBackfillFetcher[] {mapreduceBackfillFetcher, sparkBackfillFetcher, tezBackfillFetcher};
+        for (DummyElephantBackfillFetcher backfillFetcher : backfillFetchers) {
           try {
-            verify(backfillFetcher).
-                fetchJobsForBackfill(backfillTs - ElephantRunner.BACKFILL_BUFFER_TIME, lowestTsFromRM);
+            assertTrue("Jobs for backfill (of type " + backfillFetcher.getFetcherName() + ") should have been "
```

Comment:
I have mentioned it in the PR description (point 5). When running with jacoco plugin, mockito when-thenReturn/doReturn constructs lead to an error. The tests pass while running "sbt test" though. That is why build passed when Backfill PR's Travis CI run was done.
I cannot separate it out because without these changes, this specific PRs' Travis CI run would fail.

Reply:
Is when-thenReturn/doReturn constructs is just used in backfill code ?

Reply:
ok my only consideration was if it (when-thenReturn/doReturn)  was used some other way , in the some other code , which could have avoided the code changes in Back fill code. 
Ok for me as its not breaking anything else in some other code.

Reply:
Nope. None of the other tests failed. Probably they were using the construct which jacoco plugin likes :)